src/main/resources/db/migration/V202409261405__add_indexes_to_feedback_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE INDEX IF NOT EXISTS idx_feedback_employee_id ON feedback (employee_id);
+
+CREATE INDEX IF NOT EXISTS idx_feedback_created_at ON feedback (created_at);
+
+CREATE INDEX IF NOT EXISTS idx_feedback_is_anonymous ON feedback (is_anonymous);
+
+CREATE INDEX IF NOT EXISTS idx_feedback_department ON feedback (department);
```

Comment:
I would drop the department index too, I'm not sure how useful it is going to be